test/suites/scheduling/suite_test.go

```diff
@@ -644,7 +644,7 @@ var _ = Describe("Scheduling", Ordered, ContinueOnFailure, func() {
 				},
 			})
 			env.ExpectCreated(nodePool, nodeClass, pod)
-			node := env.EventuallyExpectNodeCount("==", 1)[0]
```

Comment:
I'm assuming this is because the label won't appear until CCM registers it?

Reply:
The label that I saw failures due to is actually one we add, but its added at registration time, not via user data. This is because it's added via a requirement, not a label.

Reply:
I imagine we could run into the same issue with the CCM added label though.